orquesta/events.py

```diff
@@ -37,6 +37,8 @@
 WORKFLOW_PAUSED_WORKFLOW_DORMANT = 'workflow_paused_workflow_dormant'
 WORKFLOW_RESUMING = 'workflow_resuming'
 WORKFLOW_RESUMING_WORKFLOW_COMPLETED = 'workflow_resuming_workflow_completed'
+WORKFLOW_RERUNNING = 'workflow_rerunning'
+WORKFLOW_RERUNNING_WORKFLOW_COMPLETED = 'workflow_rerunning_workflow_completed'
```

Comment:
Remove if these events are no longer needed.  If that's the case, rebase the PR to not have a commit to add/delete them.